### PR TITLE
GA 4.80.0 change logs 

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
 ## 4.80.0 (Unreleased)
+UNKNOWN CHANGELOG TYPE:
```
Comment (slevenick) [inline CHANGELOG.md:2]: Most of these look like enhancements?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,45 @@
...
+* Add workflow for issue labeler backfill ([#15552](https://github.com/hashicorp/terraform-provider-google/pull/15552))
+* Allow to remove the networks block in the resource google_dns_response_policy ([#15557](https://github.com/hashicorp/terraform-provider-google/pull/15557))
+* Update RouterBgpPeer.yaml ([#15541](https://github.com/hashicorp/terraform-provider-google/pull/15541))
+NOTES:
+* alloydb: fixed import path doc for `google_alloydb_instance` ([#15578](https://github.com/hashicorp/terraform-provider-google/pull/15578))
```
Comment (slevenick) [inline CHANGELOG.md:7]: Doc-only?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,45 @@
...
+DEPRECATIONS:
+* dataplex: deprecated the following `google_dataplex_datascan` fields: `dataProfileResult` and `dataQualityResult` ([#15528](https://github.com/hashicorp/terraform-provider-google/pull/15528))
+* firebase: deprecated `google_firebase_project_location` in favor of `google_firebase_storage_bucket` and `google_firestore_database` ([#15526](https://github.com/hashicorp/terraform-provider-google/pull/15526))
+
+FEATURES:
+* **New Data Source:** `google_sql_database_instance_latest_recovery_time` ([#15551](https://github.com/hashicorp/terraform-provider-google/pull/15551))
+* **New Resource:** `google_certificate_manager_trust_config` ([#15562](https://github.com/hashicorp/terraform-provider-google/pull/15562))
+* **New Resource:** `google_compute_region_security_policy_rule` ([#15523](https://github.com/hashicorp/terraform-provider-google/pull/15523))
+* **New Resource:** `google_gke_hub_membership_rbac_role_binding` ([#15544](https://github.com/hashicorp/terraform-provider-google/pull/15544))
+* **New Resource:** `google_iam_deny_policy` (ga only) ([#15571](https://github.com/hashicorp/terraform-provider-google/pull/15571))
```
Comment (slevenick) [inline CHANGELOG.md:17]:
```suggestion
* **New Resource:** `google_iam_deny_policy` ([#15571](https://github.com/hashicorp/terraform-provider-google/pull/15571))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,45 @@
...
+* **New Resource:** dataform_repository_workflow_config (beta) ([#15543](https://github.com/hashicorp/terraform-provider-google/pull/15543))
```
Comment (slevenick) [inline CHANGELOG.md:18]: Remove beta only

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,45 @@
...
+* **New Resource:** google_bigquery_bi_reservation ([#15527](https://github.com/hashicorp/terraform-provider-google/pull/15527))
+
+IMPROVEMENTS:
+* alloydb: added `restore_backup_source` and `restore_continuous_backup_source` fields to support restore feature in `google_alloydb_cluster` resource. ([#15580](https://github.com/hashicorp/terraform-provider-google/pull/15580))
+* artifactregistry: added `cleanup_policies` and `cleanup_policy_dry_run` fields to resource `google_artifact_registry_repository` ([#15561](https://github.com/hashicorp/terraform-provider-google/pull/15561))
+* clouddeploy: promoted `multi_target` to GA in `google_clouddelploy_target` ([#15564](https://github.com/hashicorp/terraform-provider-google/pull/15564))
```
Comment (slevenick) [inline CHANGELOG.md:24]: "added `multi_target` in ..."

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,45 @@
...
+* compute: added `security_policy` field to `google_compute_target_instance` resource (beta) ([#15566](https://github.com/hashicorp/terraform-provider-google/pull/15566))
```
Comment (slevenick) [inline CHANGELOG.md:25]: Remove

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,45 @@
...
+* compute: added support for specifying regional disks for `google_compute_instance` `boot_disk.source` ([#15597](https://github.com/hashicorp/terraform-provider-google/pull/15597))
+* container: added `additional_pod_ranges_config` field to `google_container_cluster` resource ([#15600](https://github.com/hashicorp/terraform-provider-google/pull/15600))
+* containeraws: made `config.labels` updatable in `google_container_aws_node_pool` ([#15564](https://github.com/hashicorp/terraform-provider-google/pull/15564))
+* dataplex: added fields `data_profile_spec.post_scan_actions`, `data_profile_spec.include_fields` and `data_profile_spec.exclude_fields` ([#15545](https://github.com/hashicorp/terraform-provider-google/pull/15545))
+* firebase: added `api_key_id` field to `google_firebase_web_app`, `google_firebase_android_app`, and `google_firebase_apple_app`. ([#15577](https://github.com/hashicorp/terraform-provider-google/pull/15577))
+* gkehub: promoted the `google_gke_hub_feature_membership` resource to GA ([#15604](https://github.com/hashicorp/terraform-provider-google/pull/15604))
```
Comment (slevenick) [inline CHANGELOG.md:33]: Reword

**Review by slevenick — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,45 @@
...
+* firebase: deprecated `google_firebase_project_location` in favor of `google_firebase_storage_bucket` and `google_firestore_database` ([#15526](https://github.com/hashicorp/terraform-provider-google/pull/15526))
+
+FEATURES:
+* **New Data Source:** `google_sql_database_instance_latest_recovery_time` ([#15551](https://github.com/hashicorp/terraform-provider-google/pull/15551))
+* **New Resource:** `google_certificate_manager_trust_config` ([#15562](https://github.com/hashicorp/terraform-provider-google/pull/15562))
+* **New Resource:** `google_compute_region_security_policy_rule` ([#15523](https://github.com/hashicorp/terraform-provider-google/pull/15523))
+* **New Resource:** `google_gke_hub_membership_rbac_role_binding` ([#15544](https://github.com/hashicorp/terraform-provider-google/pull/15544))
+* **New Resource:** `google_iam_deny_policy` ([#15571](https://github.com/hashicorp/terraform-provider-google/pull/15571))
+* **New Resource:** dataform_repository_workflow_config (beta) ([#15543](https://github.com/hashicorp/terraform-provider-google/pull/15543))
```
Comment (ScottSuarez) [inline CHANGELOG.md:18]:
```suggestion
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,4 +1,44 @@
...
+* **New Resource:** `google_compute_region_security_policy_rule` ([#15523](https://github.com/hashicorp/terraform-provider-google/pull/15523))
+* **New Resource:** `google_gke_hub_membership_rbac_role_binding` ([#15544](https://github.com/hashicorp/terraform-provider-google/pull/15544))
+* **New Resource:** `google_iam_deny_policy` ([#15571](https://github.com/hashicorp/terraform-provider-google/pull/15571))
+* **New Resource:** google_bigquery_bi_reservation ([#15527](https://github.com/hashicorp/terraform-provider-google/pull/15527))
+
+IMPROVEMENTS:
+* alloydb: added `restore_backup_source` and `restore_continuous_backup_source` fields to support restore feature in `google_alloydb_cluster` resource. ([#15580](https://github.com/hashicorp/terraform-provider-google/pull/15580))
+* artifactregistry: added `cleanup_policies` and `cleanup_policy_dry_run` fields to resource `google_artifact_registry_repository` ([#15561](https://github.com/hashicorp/terraform-provider-google/pull/15561))
+* clouddeploy: promoted `multi_target` to GA in `google_clouddelploy_target` ([#15564](https://github.com/hashicorp/terraform-provider-google/pull/15564))
```
Comment (ScottSuarez) [inline CHANGELOG.md:23]:
```suggestion
* clouddeploy: added `multi_target` to in `google_clouddelploy_target` ([#15564](https://github.com/hashicorp/terraform-provider-google/pull/15564))
```

File: CHANGELOG.md (hunk 2 above)
Comment (ScottSuarez) [reply CHANGELOG.md]: removed

**Review by slevenick — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,4 +1,39 @@
 ## 4.80.0 (Unreleased)
+DEPRECATIONS:
+* dataplex: deprecated the following `google_dataplex_datascan` fields: `dataProfileResult` and `dataQualityResult` ([#15528](https://github.com/hashicorp/terraform-provider-google/pull/15528))
+* firebase: deprecated `google_firebase_project_location` in favor of `google_firebase_storage_bucket` and `google_firestore_database` ([#15526](https://github.com/hashicorp/terraform-provider-google/pull/15526))
+
+FEATURES:
+* **New Data Source:** `google_sql_database_instance_latest_recovery_time` ([#15551](https://github.com/hashicorp/terraform-provider-google/pull/15551))
+* **New Resource:** `google_certificate_manager_trust_config` ([#15562](https://github.com/hashicorp/terraform-provider-google/pull/15562))
+* **New Resource:** `google_compute_region_security_policy_rule` ([#15523](https://github.com/hashicorp/terraform-provider-google/pull/15523))
+* **New Resource:** `google_gke_hub_membership_rbac_role_binding` ([#15544](https://github.com/hashicorp/terraform-provider-google/pull/15544))
```
Comment (ScottSuarez) [inline CHANGELOG.md:10]:
```suggestion
```

hotfixing removal of this resource

Comment (ScottSuarez) [reply CHANGELOG.md:10]: https://github.com/GoogleCloudPlatform/magic-modules/pull/8776

Comment (ScottSuarez) [reply CHANGELOG.md:10]: https://github.com/hashicorp/terraform-provider-google/commit/feb52d5adb7591bff1b7932471fe82e2feb4d2f7

**Comment by ScottSuarez** [comment]
Cherry picked resource removal. Was a beta only resource.
https://github.com/hashicorp/terraform-provider-google/commit/feb52d5adb7591bff1b7932471fe82e2feb4d2f7

Already updated release notes to reflect.

**Pull request merged**